doc/source/install.rst

```diff
@@ -207,7 +207,7 @@ installed), make sure you have `pytest
 
 ::
 
-    >>> import pandas as pd
+    >>> import pandas as pd  # noqa: F811
```

Comment:
same

Reply:
On this I agree with you. I can remove the explicit import.